src/main/java/com/taskmanagement/history/HistoryService.java

```diff
@@ -0,0 +1,50 @@
+package com.taskmanagement.history;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class HistoryService implements IHistoryService{
+    @Autowired
+    private IHistoryRepo historyRepo;
+
+    @Override
+    public Iterable<History> findAll() {
+        return historyRepo.findAll();
+    }
+    @Override
+    public List<HistoryDto> findAllDto() {
+        List<HistoryDto> historyDtos = new ArrayList<>();
```

Comment:
I cannot get your meaning about name of this variable 'historyDtos '. When you create variables, functions..., please choose a specific name that helps another person can understand it.